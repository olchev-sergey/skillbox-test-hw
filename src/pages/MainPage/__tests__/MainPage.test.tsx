import { render } from '@testing-library/react'
import { MainPage } from '../MainPage'

describe('Тест компонента MainPage', () => {
  it('Проверка корректного отображения содержимого страницы', () => {
    const { container } = render(<MainPage />)

    expect(container).toMatchSnapshot()
  })
})
