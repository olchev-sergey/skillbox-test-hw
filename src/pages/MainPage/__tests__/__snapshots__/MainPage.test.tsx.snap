// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тест компонента MainPage Проверка корректного отображения содержимого страницы 1`] = `
<div>
  <div
    class="mainPage"
  >
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nam cumque eligendi nobis reprehenderit delectus veritatis fuga ipsa vitae aliquid. Totam dicta quis iusto vero nisi et ex cupiditate corporis architecto?
    </p>
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Exercitationem culpa assumenda laudantium libero sapiente vitae ut maiores quod ad, ducimus eius dicta et, odio, beatae maxime voluptatem officiis unde molestiae.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi accusamus, corporis suscipit non quaerat aspernatur maxime voluptatum totam voluptates omnis saepe similique aut earum dolor necessitatibus quia quam, porro architecto!
    </p>
  </div>
</div>
`;

exports[`Тест компонента MainPage Проверка корректного рендеринга страницы 1`] = `
<div>
  <div
    class="mainPage"
  >
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nam cumque eligendi nobis reprehenderit delectus veritatis fuga ipsa vitae aliquid. Totam dicta quis iusto vero nisi et ex cupiditate corporis architecto?
    </p>
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Exercitationem culpa assumenda laudantium libero sapiente vitae ut maiores quod ad, ducimus eius dicta et, odio, beatae maxime voluptatem officiis unde molestiae.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi accusamus, corporis suscipit non quaerat aspernatur maxime voluptatum totam voluptates omnis saepe similique aut earum dolor necessitatibus quia quam, porro architecto!
    </p>
  </div>
</div>
`;
